CHANGELOG.md

```diff
@@ -8,6 +8,10 @@ Operators, you should copy/paste content of this content straight to your projec
 
 If you were at `firehose-core` version `1.0.0` and are bumping to `1.1.0`, you should copy the content between those 2 version to your own repository, replacing placeholder value `fire{chain}` with your chain's own binary.
 
+## Unreleased
+
+* Added `--merger-delete-threads` to customize the number of threads the merger will use to delete files.
```

Comment:
Can you add a note that this is particularly useful for those on Ceph storage provider.

Reply:
And how manu threads you use there